.eslintrc.js

```diff
@@ -20,6 +20,12 @@ module.exports = {
         'node/global-require': 'off',
       },
     },
+    {
+      files: '*.md/*.js',
+      parserOptions: {
+        sourceType: 'module',
+      },
```

Comment:
Workaround since the top-level repository is not using pure ES modules yet.
To remove once this is not the case anymore.